libs/category/state/src/reducers/category/pure/build-metadata-from-request.ts

```diff
@@ -1,9 +1,9 @@
 import {
   DaffCategoryPageMetadata,
-  DaffCategoryRequest,
+  DaffCategoryIdRequest,
 } from '@daffodil/category';
 
-type MetadataRequestIntersection = keyof DaffCategoryPageMetadata & keyof DaffCategoryRequest;
+type MetadataRequestIntersection = keyof DaffCategoryPageMetadata & keyof DaffCategoryIdRequest;
 
 const allowedMetadataKeys: MetadataRequestIntersection[] = [
   'id',
```

Comment:
In retrospect, is `id` necessary?

Reply:
Probably not, and that will allow the function to be typed on the request union.

Reply:
changed